project-base/src/Shopsys/ShopBundle/Controller/Styleguide/StyleguideController.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\ShopBundle\Controller\Styleguide;
+
+use Shopsys\ShopBundle\Controller\Front\FrontBaseController;
+
+class StyleguideController extends FrontBaseController
+{
+    public function styleguideAction()
```

Comment:
I am missing return typehint

Reply:
We don't use return typehints in controllers... I don't really know why :smile: